src/DependencyInjection/DoctrinePHPCRExtension.php

```diff
@@ -487,12 +487,12 @@ private function loadOdmDocumentManager(array $documentManager, ContainerBuilder
     /**
      * {@inheritdoc}
      */
-    protected function getMappingDriverBundleConfigDefaults(array $bundleConfig, \ReflectionClass $bundle, ContainerBuilder $container)
-    {
-        $this->bundleDirs[] = \dirname($bundle->getFileName());
-
-        return parent::getMappingDriverBundleConfigDefaults($bundleConfig, $bundle, $container);
-    }
+//    protected function getMappingDriverBundleConfigDefaults(array $bundleConfig, \ReflectionClass $bundle, ContainerBuilder $container)
+//    {
+//        $this->bundleDirs[] = \dirname($bundle->getFileName());
+//
+//        return parent::getMappingDriverBundleConfigDefaults($bundleConfig, $bundle, $container);
```

Comment:
do you need to remove this for symfony 6 to work, or can we re-enable it?

Reply:
Hmm yes, this was something I commented this so I could continue testing. I think it should be renabled to keep the old behavior but all the tests still pass without this so I'm not sure if this is needed then 🤔 

https://github.com/symfony/symfony/blob/80f4e7628511fafb38c5b2e4fbd85e264c247793/src/Symfony/Bridge/Doctrine/DependencyInjection/AbstractDoctrineExtension.php#L140

The issue here is that symfony 6 added a union return type `array|false` to `getMappingDriverBundleConfigDefaults` but I don't think we can support this easily. Even with a seperate class for old and newer version this would still cause a syntax error on sf <6

So I'm not sure how to move this further

Reply:
The more I look at it, it seems that the `$this->bundleDirs` property is never used 🤔

Reply:
i was looking at all the old symfony versions, even 2.8 but can not find any that would define that bundleDirs variable.

and actually the property is private, so as it is not used inside this class, it can't be needed. glad if you can remove the method (so that the parent method is called directly) and also delete the unused property.

Reply:
Done!